docs/installation/linux/debian.md

```diff
@@ -77,7 +77,7 @@ from the new repository:
 
  5. Add the new `GPG` key.
 
-         $ apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+         $ apt-key adv --keyserver hkp://pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
```

Comment:
Something is definitely up here -- `pool.sks-keyservers.net` is a rotating set of servers, and not all of them support port 80 (that's what the `p80` pool is for).  We used to use the `ha` pool to insulate ourselves a bit more from transitive failures, but too many users were having firewall issues trying to access it so we had to switch to the `p80` pool (and thus port 80) instead.